client/apps/edit/components/content/index.jsx

```diff
@@ -1,13 +1,12 @@
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { EditArticle } from './article_layouts/article.jsx'
+import EditArticle from './article_layouts/article.jsx'
```

Comment:
how come we include the file extension in those imports?

Reply:
Good catch! removing